kong/plugins/moesif/schema.lua

```diff
@@ -1,8 +1,9 @@
 return {
   fields = {
     api_endpoint = {required = true, type = "url", default = "https://api.moesif.net"},
-    timeout = {default = 10000, type = "number"},
-    keepalive = {default = 10000, type = "number"},
+    timeout = {default = 1000, type = "number"},
```

Comment:
i feel like we should have two timeouts.. 1) connection timeout, which should be at max a second. 2) send timeout..depends on the payload size, it could take more time, so may be 2 seconds.

Reply:
keepalive make 5 seconds

Reply:
agree